spec/models/item_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+describe Item do
+
+  before(:each) do
+    Item.find_or_create_by(name: "socks", description: "they come in pairs", price: 10, email: "someone@gmail.com")
+    @item = Item.first
```

Comment:
Since you have DatabaseCleaner, you should assume that this item does not exist. No need for `find_or_create`.